spec/formatting.md

```diff
@@ -372,6 +372,18 @@ to provide the _pattern_ for the formatting operation.
 This is done by ordering and filtering the available _variant_ statements
 according to their _key_ values and selecting the first one.
 
+> [!NOTE]
+> At least one _variant_ is required to have all of its _keys_ consist of
+> the fallback value `*`.
+> Some _selectors_ might be implemented in a way that this _variant_
+> cannot be selected in a _valid_ _message_.
+> In other cases, this _variant_ might be unreachable only in certain locales.
```

Comment:
I'm not sure the note about locales is relevant here? How about:

```suggestion
> In other cases, this _variant_ may be interpreted as a valid catch-all variant, rather than a fallback in case of failed selection.
```